pkg/accounting/accounting.go

```diff
@@ -216,6 +221,65 @@ func (a *Accounting) getPeerBalance(peer swarm.Address) (*PeerBalance, error) {
 	return peerBalance, nil
 }
 
+func (a *Accounting) Balances() (map[string]int64, error) {
+	peersBalances := make(map[string]int64)
+
+	a.balancesMu.Lock()
+	for peer, balance := range a.balances {
+		peersBalances[peer] = balance.balance
+	}
+	a.balancesMu.Unlock()
+
+	err := a.CompleteFromStore(peersBalances)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return peersBalances, nil
+}
+
+func (a *Accounting) CompleteFromStore(s map[string]int64) error {
```

Comment:
why is this method exported? also, it is not clear to me from the method name (and the fact that it is not documented) what this function is meant to do